src/inmanta/parser/chache.py

```diff
@@ -0,0 +1,100 @@
+"""
```

Comment:
I didn't notice before but this file is named `chache.py` (with an additional "h").

Reply:
dyslexia (fixed)